src/stats.jl

```diff
@@ -279,7 +282,7 @@ Calculate the mean of a chain.
 """
 function mean(chains::Chains; kwargs...)
     # Store everything.
-    funs = [mean∘cskip]
+    funs = [mean]
```

Comment:
Unrelated:
```suggestion
    funs = [mean∘cskip]
```

Reply:
This is related, `cskip` doesn't work with weights.

Reply:
See my comment above.